### DISCUSSION TITLE
Is it possible to fix output of rpn?

### DISCUSSION THREAD

**Discussion by babajakpum** [open-mmlab/mmdetection#11578] [Q&A]
Hello, all!

I am new to this area and mmdetection, so please kindly understand if I don't get your advice. 

During the inference, is it possible to make rpn always give one bbox that covers whole image?
For example, proposed bbox(or roi) : [0,0,image width, image height].

I know that output of rpn is in form of feature, not bbox format. So I wonder if above is possible.

Thanks a lot!

**Comment by PushpakBhoge** [comment]
Yes if that is your requirement you can do it you need to provide proposal externall as a file duting training. else you can write a class which inherit from the Rpnhead class then you can modify the functionality
but I would not recommend doing that
